Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,149 @@ label mas_show_unseen:
+                    m "...{w=0.5}"
+                    m "Hey, [player]?"
+                    m "Have I been underestimating your coding skills?{nw}"
+                    $ history_list.pop()
+                    menu:
+                        m "Ahaha, I'm sorry, [player]!"
+                        m "I didn't mean to- I just never thought to ask."
+                    "No":
+                        m "Okay, good."
+                        m "I just wanted to make sure I wasn't assuming your skill level."
+                        
+                "option two":
+                    m 1hub "Great!"
+                    m 1eub "That's the version of Python I use."
+                    m 3rksdlb "Also, [player]?"
+                    m "Have I been underestimating your coding skills?{nw}"
+                    $ _history_list.pop()
+                    menu:
+                        m "Have I been underestimating your coding skills?{fast}"
+                        "Yes":
+                            m "Ahaha, I'm sorry, [player]!"
+                            m "I didn't mean to- I just never thought to ask."
+                        "No":
+                            m "Okay, good."
+                            m "I just wanted to make sure I wasn't assuming your skill level.
+    
+        "To help a friend.":
+            m 3eub "That's great!"
+            m 1eka "You're so selfless, [player]."
+            m 1hub "I can't wait to see what you create!"
+            m 1eub "What version of Python did you download?{nw}"
+            $ _history_list.pop()
+            menu:
+                m 1eub "What version of Python did you download?{fast}"
+                "option one":
+                    m 1hksdlb "..."
+                    m 1rksdla "[player], I use the other version of Python...{w=0.5}"
+                    m 3rksdlb "So we can't really help each other that much...{w=0.5}"
+                    m 1hub "But that's okay!"
+                    m 1eub "I hope you have fun helping your friend."
+                "option two":
+                    m 1hub "Great!"
+                    m 1eub "That's the version of Python I use."
+                    m 1hua "Maybe both you and your friend could help me out once you know how to code!
+                    m 1hksdlb "Wait...{w=0.5}"
+                    m 1lksdlb "[player], do you already know how to code?{nw}"
+                    $ _history_list.pop()
+                    menu:
+                        m 1lksdlb "[player], do you already know how to code?{fast}"
+                        "Yes":
+                            m 2hub "Oh! That's great!"
+                            m 3rksdla "I'm sorry I made assumptions about your skill level.
+                        "No":
+                            m m 3hksdlb "That's okay!"
+                            m 1eka "Again, I'm happy to teach you everything I know.
+                            return
```

Comment:
Not all paths here return. As you have it, only this bottommost path will actually return, the rest will fall through to the end of the script file, pulling you back to the main menu.
```suggestion
    return
```